source/Playnite/Localization/LocSource.xaml

```diff
@@ -762,21 +762,48 @@ Version {2}
     
 {0}</sys:String>
 
+    <!--Battle.net-->
+    <sys:String x:Key="LOCBattleNetLinksPCGW">PCGamingWiki</sys:String>
+
+    <!--Bethesda-->
+    <sys:String x:Key="LOCBethesdaLinksPCGW">PCGamingWiki</sys:String>
+
     <!--itch.io-->
     <sys:String x:Key="LOCItchioSignInNotif">itch.io client will now open. Please sign in and then close this message.</sys:String>
     <sys:String x:Key="LOCItchioSignInWaitMessage">Waiting for user to sign in, please close this when you're done…</sys:String>
     <sys:String x:Key="LOCItchioClientNotInstalledError">Itch client version 25 or newer is not installed.</sys:String>
+    <sys:String x:Key="LOCItchioLinksPCGW">PCGamingWiki</sys:String>
 
     <!--GOG-->
     <sys:String x:Key="LOCGOGUseLogin">Account Login</sys:String>
     <sys:String x:Key="LOCGOGUseAccountName">Account Name</sys:String>
     <sys:String x:Key="LOCGOGAccountSyncDescription" xml:space="preserve">Login to your account or use your public account name to sync your full library.
 Note that the Account Name option only works for accounts whose game libraries have been set in GOG's privacy settings to be publicly visible.</sys:String>
+    <sys:String x:Key="LOCGOGLinksPCGW">PCGamingWiki</sys:String>
+    <sys:String x:Key="LOCGOGLinksForum">Forum</sys:String>

```

Comment:
Please don't create separate string for each library. "Forum" (and other strings like Store) is pretty universal and I don't see a point of creating separate variants of it for each library provider.